docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -40,25 +40,26 @@ If you prefer a step-by-step installation process, complete the following steps
 
     ```bash
     sudo apt-get update
-    sudo apt-get install ca-certificates curl apt-transport-https lsb-release gnupg
+    sudo apt-get install apt-transport-https ca-certificates curl gnupg2 lsb-release
     ```
 
 2. Download and install the Microsoft signing key:
 
     ```bash
-    sudo mkdir -p /etc/apt/keyrings
-    curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
-        gpg --dearmor |
-        sudo tee /etc/apt/keyrings/microsoft.gpg > /dev/null
-    sudo chmod go+r /etc/apt/keyrings/microsoft.gpg
+    curl -fssL https://packages.microsoft.com/keys/microsoft.asc |\
+      gpg --dearmor -o /usr/share/keyrings/microsoft.gpg
```

Comment:
Any reason to change it to `/usr/share/keyrings/microsoft.gpg`? It seems that the previous location is correct.
> The recommended locations for keyrings are [/usr/share/keyrings](file://usr/share/keyrings) for keyrings managed by
           packages, and /etc/apt/keyrings for keyrings managed by the system operator.   ---https://manpages.ubuntu.com/manpages/jammy/en/man5/sources.list.5.html

Reply:
old habit, will change back to `/etc/apt/keyrings`